src/database/schema/ChatSchema.ts

```diff
@@ -191,7 +191,7 @@ class ChatSchema implements BaseDbSchema {
       ON A.user_id = B.user_id
       INNER JOIN ${ChannelListSchema.getTableName()} C
       ON A.channel_id = C.id
-      WHERE A.status = 'pending';`;
+      WHERE A.status = 'pending' group by A.id;`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `group by` clause is used to group rows that have the same values in specified columns into aggregated data. However, it seems you are grouping by `A.id`, which is likely a unique identifier for each row. This means that each group will contain only one row, and the `group by` clause won't have any effect. If your intention was to group by another column (like `status` or `user_id`), please update the query accordingly.
```diff
- WHERE A.status = 'pending' group by A.id;`;
+ WHERE A.status = 'pending' group by A.status;`;
```
If `A.id` is indeed the correct column to group by, then the `group by` clause is unnecessary and can be removed.
```diff
- WHERE A.status = 'pending' group by A.id;`;
+ WHERE A.status = 'pending';`;
```
Please ensure that the SQL query is achieving the desired result.


<!-- This is an auto-generated comment by OSS CodeRabbit -->